engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglPortlet.java

```diff
@@ -32,11 +30,13 @@ public String getName() {
 
     @Override
     public void postInitialise(Context context) {
-        try {
-            Display.setParent(customViewPort);
-        } catch (LWJGLException e) {
-            throw new RuntimeException("Can not initialize graphics device.", e);
-        }
+        // FIXME: LWJGL 3 - reimplement
```

Comment:
Would be nice with some more details for this - what needs to be fixed? What's broken? Should we submit an issue with details then link to that issue from the TODO? What does this portlet even do?